packages/networks/src/index.ts

```diff
@@ -133,6 +133,7 @@ const all: NetworkFromSubstrate[] = [
   {
     decimals: null,
     displayName: 'Polymath',
+    genesisHash: ['12fddc9e2128b3fe571e4e5427addcb87fcaf08493867a68dd6ae44b406b39c7'],
```

Comment:
```suggestion
    genesisHash: ['0x12fddc9e2128b3fe571e4e5427addcb87fcaf08493867a68dd6ae44b406b39c7'],
```

Needs the hex prefix.

(Sorry, should have seen this in the first rounf)